api/src/main/java/module-info.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 International Business Machines Corp. and others
+ * 
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Licensed under the Apache License, 
+ * Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+module jakarta.batch {
+
+    // Used in BatchRuntime
+    requires java.logging;
+
+    // Optional - note this transitively requires jakarta.inject
+    requires static jakarta.cdi;
```

Comment:
It's OK that jakarta.cdi is option. But shouldn't we also add an explicit dependency on jakarta.inject, since that one is really required all the time to support batch properties injection?
```suggestion
    requires jakarta.inject;
```